app/src/app/screens/Dashboard/screens/Detail/layout.tsx

```diff
@@ -16,36 +16,33 @@ interface Props {
 }
 
 function DetailContainer({ title, recipe, onDownload }: Props) {
-  const { html, scss } = recipe;
   const [isCodeVisible, setIsCodeVisible] = useState(false);
 
   const handleClick = () => setIsCodeVisible(!isCodeVisible);
 
   return (
     <>
-      <div className="row space-between m-bottom-4 middle">
-        <h1 className="title">{title}</h1>
-        <a
-          href="https://github.com/Wolox/frontend-cookbook"
-          className={`row midle ${styles.githubLink}`}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          by Tincho_otaku
-        </a>
-      </div>
-      <div className={styles.detailsRecipe}>
-        <RecipePreview recipe={recipe} />
-        <div className={styles.settingsCheckbox}>
-          <input type="checkbox" id="chkButton" onClick={handleClick} />
-          <label htmlFor="chkButton" />
+      <div className={styles.upperSection}>
+        <div className="row space-between m-bottom-4 middle">
+          <h1 className="title">{title}</h1>
+          <a
+            href="https://github.com/Wolox/frontend-cookbook"
+            className={`row midle ${styles.githubLink}`}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            by Tincho_otaku
```

Comment:
Delete this please 😆  and add a TODO to add the Github user of the creator

Reply:
⚰️